frontend/src/api/k8s/hardwareProfiles.ts

```diff
@@ -33,7 +33,7 @@ export const assembleHardwareProfile = (
   namespace: string,
   visibility: string[] = [],
 ): HardwareProfileKind => ({
-  apiVersion: `${HardwareProfileModel.apiGroup}/${HardwareProfileModel.apiVersion}`,
+  apiVersion: kindApiVersion(HardwareProfileModel),
```

Comment:
Converted direct usage -- it seems we were using `const model = { ... } satisfies K8sCommonModel` to get around this issue with our linter not appreciating the type can have a missing `apiGroup`.